### PR TITLE
Quickcheck for testing

### PR DESCRIPTION
**Pull request by larskotthoff**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -112,6 +112,7 @@ Suggests:
     pls,
     PMCMR (>= 4.1),
     pROC (>= 1.8),
+    quickcheck,
```
Comment (mllg) [inline DESCRIPTION]: quickcheck is not on CRAN, I don't think that we can put it into the DESCRIPTION w/o getting into trouble on the next release.

**Comment by berndbischl** [comment]
would better bounds for the test be "best" and "worst" from the measure?
otherwise i think we could merge this

**Comment by larskotthoff** [comment]
I didn't use `best` and `worst` in the spirit of having a hand-coded example. The `best` and `worst` values could be wrong.

**Comment by berndbischl** [comment]
you suck

**Pull request merged**